config/checker_guideline_map.json

```diff
@@ -0,0 +1,360 @@
+      "sei-cert": ["err61-cpp"]
+    },
+    "cert-fio38-c": {
+      "sei-cert": ["fio38-c"]
+    },
+    "cert-flp30-c": {
+      "sei-cert": ["flp30-c"]
+    },
+    "cert-msc30-c": {
+      "sei-cert": ["msc30-c", "msc50-cpp"]
+    },
+    "cert-msc32-c": {
+      "sei-cert": ["msc32-c", "msc51-cpp"]
+    },
+    "cert-msc50-cpp": {
+      "sei-cert": ["msc50-cpp", "msc30-c"]
+    },
+    "cert-msc51-cpp": {
+      "sei-cert": ["msc51-cpp"]
+    },
+    "cert-mem57-cpp": {
+      "sei-cert": ["mem57-cpp"]
+    },
+    "cert-oop11-cpp": {
+      "sei-cert": ["oop11-cpp"]
+    },
+    "cert-oop54-cpp": {
+      "sei-cert": ["oop54-cpp"]
+    },
+    "cert-oop57-cpp": {
+      "sei-cert": ["oop57-cpp"]
+    },
+    "cert-oop58-cpp": {
+      "sei-cert": ["oop58-cpp"]
+    },
+    "cert-pos44-c": {
+      "sei-cert": ["pos44-c"]
+    },
+    "cert-str34-c": {
+      "sei-cert": ["str34-c"]
+    },
+    "core.CallAndMessage": {
+      "sei-cert": ["dcl41-c", "exp30-c", "exp33-c", "exp34-c", "exp39-c",
+                   "exp50-cpp", "exp53-cpp", "exp54-cpp", "exp57-cpp"]
+    },
+    "core.DivideZero": {
+      "sei-cert": ["int33-c"]
+    },
+    "core.NonNullParamChecker": {
+      "sei-cert": ["exp34-c", "exp34-c", "mem34-c"]
+    },
+    "core.NullDereference": {
+      "sei-cert": ["arr30-c", "dcl38-c"]
+    },
+    "core.StackAddressEscape": {
+      "sei-cert": ["dcl30-c", "exp54-cpp"]
+    },
+    "core.UndefinedBinaryOperatorResult": {
+      "sei-cert": ["exp33-c", "exp36-c", "exp53-cpp", "int32-c", "int34-c",
+                   "oop53-cpp"]
+    },
+    "core.uninitialized.Assign": {
+      "sei-cert": ["oop54-cpp"]
+    },
+    "core.uninitialized.UndefReturn": {
+      "sei-cert": ["oop53-cpp"]
+    },
+    "core.VLASize": {
+      "sei-cert": ["arr32-c"]
+    },
+    "cplusplus.InnerPointer": {
+      "sei-cert": ["mem50-cpp", "str52-cpp"]
+    },
+    "cplusplus.Move": {
+      "sei-cert": ["exp63-cpp"]
+    },
+    "cplusplus.NewDelete": {
+      "sei-cert": ["exp54-cpp", "mem50-cpp", "mem51-cpp", "oop54-cpp"]
+    },
+    "cplusplus.NewDeleteLeaks": {
+      "sei-cert": ["exp62-cpp", "mem51-cpp"]
+    },
+    "cppcoreguidelines-pro-type-const-cast": {
+      "sei-cert": ["exp55-cpp"]
+    },
+    "cppcoreguidelines-slicing": {
+      "sei-cert": ["oop51-cpp"]
+    },
+    "misc-assert-side-effect": {
+      "sei-cert": ["pre31-c"]
+    },
+    "misc-non-copyable-objects": {
+      "sei-cert": ["fio38-c"]
+    },
+    "misc-throw-by-value-catch-by-reference": {
+      "sei-cert": ["err61-cpp"]
+    },
+    "misc-use-after-move": {
+      "sei-cert": ["exp63-cpp"]
+    },
+    "optin.cplusplus.VirtualCall": {
+      "sei-cert": ["oop50-cpp"]
+    },
+    "optin.portability.UnixAPI": {
+      "sei-cert": ["mem30-c"]
+    },
+    "security.FloatLoopCounter": {
+      "sei-cert": ["flp30-c"]
+    },
+    "security.insecureAPI.gets": {
+      "sei-cert": ["str31-c"]
+    },
+    "security.insecureAPI.strcpy": {
```

Comment:
Remove all these as the usage of stcpy is irrelevant for this concrete rules. The analyze cannot model the size of the buffers properly

MEM30-C-B.c - use after free error
MEM34-C-A.c - free of non dynamic memory
STR31-C - is more a taint attack